docs/source/admin_guide/breaking-upgrade.md

```diff
@@ -0,0 +1,174 @@
+# Upgrade - Redeployment for Breaking Changes
+
+For versions that are known to require redeployment when upgrading, this is an outline of the steps required to upgrade QHub:
+
+- Back up user data by following [this backup guide](./backup.md).
+- Change the existing cluster to a different URL (for example `qhub-old.mycompany.com`) so it is hidden.
+- Run `qhub upgrade` to make recommended modifications to the `qhub-config.yaml` file.
+- Deploy a new QHub with the desired version (to your original preferred URL for example `qhub.mycompany.com` but a new project_name to avoid resource name clashes).
+- Restore user data to the new cluster.
+- Once you have confirmed the backup and upgrade have been successfully completed, you can destroy the old cluster.
+
+Full details of the steps appear below.
+
+Please always check [release notes](../dev_guide/changelog.md) for more details - and in all cases, backup your data before upgrading.
+
+> The rest of this guide assumes you are upgrading from version v0.3.14 to v0.4.
```

Comment:
Is this always true or can folks upgrade from any `v0.3.x`? if so it would be best to replace with that

Reply:
Will add 'or earlier'

Reply:
Will add 'or earlier'